UnrealEngine/README.md

```diff
@@ -68,16 +67,54 @@ To make this process easier, there is a new solution named JSBSimForUnreal.sln a
  - It will take care of making a clean build, and copy all needed files at the right location
 	 - All libs and headers in `UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Source\ThirdParty\JSBSim`
 	 - All resource files (aircrafts/engines/systems) in *UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Resources\JSBSim*
- 
-**3. [Optional] - Download HD resources**
+
+### 3. Build JSBSim for Unix (Macos, Linux on its way)
+
+The first step is to build JSBSim on your target platform using cmake. Open an terminal and type the following commands
+
+```bash
+mkdir build
+cd build
+cmake ..
+make
+```
+
+```bash
+# From the build folder
+cd ..
+
+# Copy headers
+rsync -avm --include='*.h' --include='*.hpp' --include='*.hxx' -f 'hide,! */' src/ UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Include/
+
+# Copy the JSBSim library (Macos)
+mkdir -p UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Lib/Mac/
+cp -Rf build/src/libJSBSim.a UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Lib/Mac/
+
+# Copy the JSBSim library (Linux)
+mkdir -p UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Lib/Linux/
+cp -Rf build/src/libJSBSim.a UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Lib/Linux/
+
+# Copy the JSBSim library (Android)
+mkdir -p UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Lib/Android/
+cp -Rf build/src/libJSBSim.a UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/ThirdParty/JSBSim/Lib/Android/
+
+# Copy the resource files
+mkdir -p UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Resources/JSBSim
+cp -Rf aircraft UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Resources/JSBSim
+cp -Rf engine UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Resources/JSBSim
+cp -Rf systems UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Resources/JSBSim
+```
```

Comment:
Isn't this content redundant with the content of `README-Unix.md` ? Please remove one (either one).

In addition the header says "Build JSBSim for Unix (Macos, **Linux on its way**)" but my understanding is that this PR allows the UE plugin to be built on both platforms ?

Reply:
My bad, I'll fix that